deploy/docker-swarm/clickhouse-setup/otel-collector-config.yaml

```diff
@@ -69,6 +60,40 @@ receivers:
               job_name: otel-collector
 
 processors:
+  logstransform/internal:
+    operators:
+      - type: trace_parser
+        if: '"trace_id" in attributes or "span_id" in attributes'
+        trace_id:
+          parse_from: attributes.trace_id
+        span_id:
+          parse_from: attributes.span_id
+        output: remove_trace_id
+      - type: trace_parser
```

Comment:
I have encountered people using `(trace|span).id` or `(trace|span)ID`. It would be good to consider adding them as well.

Reply:
I think we can address it in a sepeate issue if we should add it by default or not, added this as it was missing from swarm.

Reply:
So intially Ankit suggested just to add `traceId` and we can ask users to change it to the following. But adding more might as well add extra overhead because it will check this for each log line, but yeah lets discuss this.